lib/utils.js

```diff
@@ -252,9 +254,9 @@ exports.files = function (dir, ext, ret) {
     .filter(ignored)
     .forEach(function (path) {
       path = join(dir, path);
-      if (statSync(path).isDirectory()) {
+      if (lstatSync(path).isDirectory()) {
         exports.files(path, ext, ret);
-      } else if (path.match(re)) {
+      } else if (path.match(re) && existsSync(path)) {
```

Comment:
Yeah true. I removed  existsSync now so it still fails if .js symlinks files are broken, but also gives better error message when trying to require it later on. To me personally both are fine as our major problem was with other files than js.